addons/arsenal/functions/fnc_scanConfig.sqf

```diff
@@ -142,7 +142,7 @@ private _putList = [];
     switch true do {
         // Rifle, handgun, secondary weapons mags
         case (
-                (getNumber (_x >> "type") in [256,512,1536,16]) &&
+                (getNumber (_x >> "type") in [TYPE_MAGAZINE_PRIMARY_AND_THROW,TYPE_MAGAZINE_SECONDARY_AND_PUT,1536,TYPE_MAGAZINE_HANDGUN_AND_GL]) &&
```

Comment:
Can't find out what 1536 is. It's 0x600 and thus falls under `MaskSlotItem`
Would be interesting to know what items fall under that.

Reply:
some rockets which are not `TYPE_MAGAZINE_SECONDARY_AND_PUT` IIRC

Reply:
The number behind there defines the `SlotCount`.. Don't really understand what it means though. Certainly not it's size.

Reply:
Can confirm that A2 inventory size was changed by the type value.

Reply:
E.g. 1x1 mag vs 2x1 mag vs 6x2 mag etc.

Reply:
yeah then that's it. This should really be implemened using binary & then... But we don't have that so meh